### ISSUE TITLE
[opkgc] Wrong dependency management on Debian

### ISSUE THREAD

**Issue by dikim33** [oscar-cluster/oscar#503]
**Reported by valleegr on 21 Dec 2008 05:06 UTC**
On Debian the following kind of dependency with a package version will fail:

```
<serverDeps>
    <requires>
      <pkg version=">=3.0.6" >ganglia-gmetad</pkg>
      <pkg version=">=3.0.6" >ganglia-gmond</pkg>
      <pkg version=">=3.0.6" >ganglia-web</pkg>
      <pkg>rrdtool</pkg>
    </requires>
</serverDeps>
```

The error is:
```
[dpkg-gencontrol: warning: can't parse dependency ganglia-gmetad (None >=3.0.6)
[INFO](INFO]) dpkg-gencontrol: error: error occurred while parsing ganglia-gmetad (None >=3.0.6), ganglia-gmond (None >=3.0.6), ganglia-web (None >=3.0.6), rrdtool, oscar-base-server
[dh_gencontrol: command returned error code 65280
[INFO](INFO]) make[*** [binary-common](1]:) Error 1
[make: *** [binary-indep](INFO]) Error 2
[dpkg-buildpackage: failure: fakeroot debian/rules binary gave error exit status 2
[ERROR](INFO]) Packages generation failed
```

**Comment by dikim33** [comment]
**Modified by valleegr on 12 Jan 2009 16:19 UTC**

**Comment by dikim33** [comment]
**Comment by valleegr on 22 Jun 2009 18:17 UTC**
(In [8589]) This check-in fixes #504 - Fix the bug when specifying a version within the 
dependencies for Debian packages.
